src/content/docs/blog/roadmap-2025.md

```diff
@@ -0,0 +1,69 @@
+  * Partial support for **Astro**, **Svelte** and **Vue** files.
+  * The ability to let configuration files extend from one another, which is especially useful in monorepo and larger organizational setups.
+  * We added countless new lint rules and miscellaneous fixes and improvements.
+ * Our [team of maintainers](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#current-members) has grown from 10 members at the start of 2024 to 18 today.
+ * We won the **Productivity Booster** award of the [OS Awards 2024](https://osawards.com/javascript/2024).
+ * We gained several new [sponsors](https://github.com/biomejs/biome#sponsors).
+ * Our VS Code extension has seen an overhaul that's currently in Pre-Release.
+ * And even though this happened after the new year, we should not neglect to mention that our IDEA plugin has seen a major update too, which is now available in the nightly channel.
+
+## 💳 Enterprise Support
+
+One more thing that we are happy to announce is that as of January 2025, we are also offering [Enterprise Support](https://biomejs.dev/enterprise) for Biome. Hopefully this will allow some of our contributors to spend more of their time and effort towards Biome!
+
+## ⏭️ Biome 2.0
+
+Right now our team is busy preparing for the Biome 2.0 release. Because our project is still run by volunteer contributors, we do not have an ETA for you. But we can share some of the goodies that will be coming:
+
+* **Plugins**. A long-requested feature, we started development of Biome plugins after an [RFC process](https://github.com/biomejs/biome/discussions/1762) that started January 2024. Biome 2.0 will feature the first fruits of this labor: Users will be able to create their own lint rules using [GritQL](https://docs.grit.io/language/overview).
+* **Domains**. [Domains](https://github.com/biomejs/biome/blob/next/.changeset/introduce_the_domains_linter_feature.md) are a configuration feature that makes it easy for users to enable or disable all rules related to a specific domain, such as React, Next.js or testing frameworks. It also allows Biome to automatically enable recommended domain-specific rules based on the dependencies listed in your `package.json`.
+* **Monorepo Support**. While support for monorepos was already improved with our `extends` feature in `biome.json`, many weak spots remained. Biome 2.0 has an improved architecture based on an internal `ProjectLayout` that should resolve most of these.
+* **Suppressions**. Biome already allowed *suppression* of linter diagnostics through the use of `// biome-ignore` suppression comments. With Biome 2.0 we're adding support for `// biome-ignore-all` and `// biome-ignore-start`/`biome-ignore-end` comments.
+* **Multi-file analysis**. Last but not least, we're adding true [Multi-file support](https://github.com/biomejs/biome/issues/3307) to Biome 2.0. This means that our lint rules will be able to query information from other files, which will enable much more powerful lint rules.
```

Comment:
I would like to add Markdown support. Works have slowly started by some external contributors. https://github.com/biomejs/biome/issues/3718